### PR TITLE
Default BookReader to 1up instead of 2up on mobile

### PR DESCRIPTION
**Pull request by nsharma123**
https://webarchive.jira.com/browse/WEBDEV-5253

### CODE REVIEW

File: src/BookReader.js
Comment (iisa) [inline src/BookReader.js:504]: :nice:

**Review by iisa — APPROVED** [review]
@nsharma123 code looks good but we cannot merge the build files into main as they get generated by release flow. can u remove those before merge? thanks!

**Comment by iisa** [comment]
@nsharma123 please remove build files before merge. thanks!

**Comment by iisa** [comment]
tests also need to be checked, they are failing

**Comment by nsharma123** [comment]
yes, I will remove those and fix the tests as well. Thanks!

File: src/BookReader.js
Comment (iisa) [inline src/BookReader.js:480]: this conditional is "[yoda style](https://eslint.org/docs/latest/rules/yoda)". we don't usually use this style in BR.

can a falsey work? `if (!this.options.defaults)` ?

File: src/BookReader.js
Comment (iisa) [inline src/BookReader.js:481]: Suggestion, potential blocker: we have possible 3rd option for thumbnail view - can you add support here for it? @nsharma123

**Review by iisa — CHANGES_REQUESTED** [review]
@nsharma123 thanks for the fix! got a couple of suggestions, I just want us to ensure we cover thumbnail mode when it is set to default. thank you!

**Review by iisa — APPROVED** [review]
thanks for making sure all modes are covered

**Comment by iisa** [comment]
looks good once tests pass

**Pull request merged**
